Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -276,30 +284,35 @@ class UpstreamIntegrationTests: TestBase, AnyCodableAsserts {
         Edge.sendEvent(experienceEvent: experienceEvent)
 
         // Verify
-        let expectedStateStoreJSON = #"""
+        let expectedStateStoreJSON = """
         {
           "payload": [
             {
-              "maxAge": 123,
               "key": "kndctr_972C898555E9F7BC7F000101_AdobeOrg_cluster",
+              "maxAge": 123,
               "value": "stringType"
             },
             {
-              "maxAge": 123,
               "key": "kndctr_972C898555E9F7BC7F000101_AdobeOrg_identity",
+              "maxAge": 123,
               "value": "stringType"
             }
           ]
         }
-        """#
+        """
 
         // See testSendEvent_receivesExpectedEventHandles for existence validation
         let stateStoreEvent = getEdgeEventHandles(expectedHandleType: TestConstants.EventSource.STATE_STORE).last!
 
-        // Exact match used here to strictly validate `payload` array element count == 2
         assertExactMatch(expected: expectedStateStoreJSON,
-                        actual: stateStoreEvent,
-                        typeMatchPaths: ["payload[0].maxAge", "payload[0].value", "payload[1].maxAge", "payload[1].value"])
+                         actual: stateStoreEvent,
+                         pathOptions:
+                            ValueTypeMatch(paths:
+                                            "payload[0].maxAge",
+                                            "payload[0].value",
+                                            "payload[1].maxAge",
+                                            "payload[1].value"),
```

Comment:
Nit: you can save couple lines with:
assertTypeMatch(expected: expectedStateStoreJSON, 
                               actual: stateStoreEvent,
                               pathOptions: ValueExactMatch(paths:
                                                                             payload[0].key,
                                                                             payload[1].key)